js/main.js

```diff
@@ -155,12 +159,20 @@ function handleSwapClick(event) {
       if (i === $dataViewNodeList.length - 1) {
         var $liNodeList = document.querySelectorAll('li');
         for (var j = 0; j < $liNodeList.length; j++) {
-          $ul.removeChild($liNodeList[j]);
+          $matchesUl.removeChild($liNodeList[j]);
         }
-        $miniMessageIcon.className = 'hidden';
+        $miniMessageIconMatch.className = 'hidden';
+        $miniMessageIconSuperlikes.className = 'hidden';
         data.ratingsInfo = {};
         showFirstNFT();
       }
+    } else if (event.target.matches('.fa-crown')) {
+      if ($dataViewNodeList[i].dataset.view !== 'superlikes') {
+        $dataViewNodeList[i].className += ' hidden';
+      } else if ($dataViewNodeList[i].dataset.view === 'superlikes') {
+        className = $dataViewNodeList[i].className;
+        $dataViewNodeList[i].className = className.replace(/ hidden/g, '');
```

Comment:
You can use the `classList.remove` method instead of restoring to RegEx.

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/remove